src/app/containers/MediaPlayer/helpers/logMissingBlockId.js

```diff
@@ -0,0 +1,12 @@
+import nodeLogger from '#lib/logger.node';
+import { MEDIA_PLAYER_STATUS } from '#lib/logger.const';
+
+const logger = nodeLogger(__filename);
+
+const logMissingBlockId = ({ url, assetType }) => {
+  const blockIdError = 'Missing Block ID';
+  const message = { url, assetType, blockIdError };
+  logger.warn(MEDIA_PLAYER_STATUS, message);
+};
+
+export default logMissingBlockId;
```

Comment:
Do we need a test for this function?

Reply:
I've added a check here to ensure it is called when there is no versionID: https://github.com/bbc/simorgh/pull/7472/files#diff-cbdd37f9faac2cd6bf9183b657587a55R62

I don't think the function itself needs a test as it is logging what is passed in and the logger itself has tests

Reply:
And ensuring it isn't called for unavailable videos: https://github.com/bbc/simorgh/pull/7472/files#diff-cbdd37f9faac2cd6bf9183b657587a55R63